generator/bin/makeCompositeTile.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+'use strict';
+var Promise = require('bluebird');
+
+var argv = require('yargs')
+    .usage('Usage: $0 \<tiles\> [options]')
+    .example('$0 batched.b3dm instanced.i3dm points.pnts -o output/composite.cmpt')
+    .alias('o', 'output')
+    .default('o', 'output/composite.cmpt')
+    .nargs('o', 1)
+    .describe('o', 'Output path where the composite tile should be written.')
+    .boolean('z')
+    .alias('z', 'gzip')
+    .default('z', false)
+    .describe('z', 'Flag to gzip the output composite tile.')
+    .help('h')
+    .alias('h', 'help')
+    .demand(1)
+    .argv;
+
+var readTile = require('../lib/readTile');
+var writeTile = require('../lib/writeTile');
+
+var tilePaths = argv._;
+var outputPath = argv.o;
+
+Promise.map(tilePaths, function(tilePath) {
```

Comment:
How about just separating out the composite logic? Make it a function that takes an array of buffers and outputs the composite tile buffer. I'd rather not force it to be to/from the filesystem.